README.md

```diff
@@ -261,19 +239,39 @@ You can configure what events will be captured by passing the following argument
 
 For example:
 ```python
-from scenario import Context, Event, State
+import scenario
 
 def test_emitted_full():
-    ctx = Context(
+    ctx = scenario.Context(
         MyCharm,
         capture_deferred_events=True,
         capture_framework_events=True,
     )
-    ctx.run("start", State(deferred=[Event("update-status").deferred(MyCharm._foo)]))
+    in_state = scenario.State(
+        deferred=[
+            defer(MyCharm.on.config_changed, MyCharm._on_config_changed)  # No second arg could be all observers?
+        ]
+    )
+    in_state = scenario.State(
+        deferred = [
+            Deferred(MyCharm.on.config_changed, MyCharm._on_config_changed)  # No second arg could be all observers?
+        ]
+    )
+    in_state = scenario.State(
+        deferred=[
+            {MyCharm.on.config_changed: [MyCharm._on_config_changed]}  # A None value could be all observers?
+        ]
+    )
+    in_state = scenario.State(
+        deferred=[
+            (MyCharm.on.config_changed, MyCharm._on_config_changed)  # Could take a CharmEvents or 2-tuples, CharmEvents meaning all observers?
+        ]
+    )
+    ctx.run(MyCharm.on.start, in_state)
 
     assert len(ctx.emitted_events) == 5
     assert [e.handle.kind for e in ctx.emitted_events] == [
-        "update_status",
+        "config_changed",
```

Comment:
I'm wondering, do we even need the `emitted_events` API? Shouldn't we be testing for side effects / what the events actually changed instead? Probably a Pietro discussion.